packages/ckeditor5-bookmark/package.json

```diff
@@ -38,7 +38,6 @@
     "@ckeditor/ckeditor5-select-all": "43.1.1",
     "@ckeditor/ckeditor5-table": "43.1.1",
     "@ckeditor/ckeditor5-theme-lark": "43.1.1",
-    "@ckeditor/ckeditor5-typing": "43.1.1",
```

Comment:
If we add `Essentials` as dependency, do we still need `Enter` and `Undo` in dependencies?

Reply:
Good catch, wille remove, I just need to change plugins to `Essentials` in editing tests.